citi-onboarding/iJunior-onepage
# Feature/frontend-footer

## What I did

- Created the front-end footer section
- Created the footer
   - Has logo and scrollTop button


| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122576557-2dc3ad80-d028-11eb-9028-8b9334968e73.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122577634-64e68e80-d029-11eb-840f-62bceeec3b4f.png"> |


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/frontend-footer`
   1. `git pull origin feature/frontend-footer`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `/client/src/views/Footer`

1. Things to change in the code during the test
   ```Footer.js
    // insert one div as the first div
   <>
        <div className="test"></div>
   </>
   ```
   ```Footer.css
   /* insert props into div test */
   .test{ height: 100vh;}
   ```
1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 1h30min

**Resolves:** # Front-end footer + scrollTop
